sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs

```diff
@@ -1884,18 +1884,18 @@ private static void ValidateHookToCreate(NotificationHook hook, string paramName
         #region Credential
 
         /// <summary>
-        /// Creates a <see cref="DataSourceCredential"/> and assigns it a unique ID. This API provides different ways of
+        /// Creates a <see cref="DataSourceCredentialEntity"/> and assigns it a unique ID. This API provides different ways of
         /// authenticating to a <see cref="DataFeedSource"/> for data ingestion when the default authentication method does not suffice.
-        /// Please see <see cref="DataSourceCredential"/> for a list of supported credentials.
+        /// Please see <see cref="DataSourceCredentialEntity"/> for a list of supported credentials.
         /// </summary>
-        /// <param name="dataSourceCredential">Specifies how the created <see cref="DataSourceCredential"/> should be configured.</param>
+        /// <param name="dataSourceCredential">Specifies how the created <see cref="DataSourceCredentialEntity"/> should be configured.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>
-        /// A <see cref="Response{T}"/> containing the result of the operation. The result is a <see cref="DataSourceCredential"/>
-        /// instance containing information about the created data source credential.
+        /// A <see cref="Response{T}"/> containing the result of the operation. The result is a <see cref="DataSourceCredentialEntity"/>
+        /// instance containing information about the created data source credential entity.
         /// </returns>
         /// <exception cref="ArgumentNullException"><paramref name="dataSourceCredential"/> is null.</exception>
-        public virtual async Task<Response<DataSourceCredential>> CreateDataSourceCredentialAsync(DataSourceCredential dataSourceCredential, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<DataSourceCredentialEntity>> CreateDataSourceCredentialAsync(DataSourceCredentialEntity dataSourceCredential, CancellationToken cancellationToken = default)
```

Comment:
does the parameter name need to change too?

Reply:
if yes, present in other places

Reply:
I think we don't need to be that strict about parameter names (I would name it only `credential` if we designed it today, to tell the truth). I'm keeping it `dataSourceCredential` to align with other languages.

Reply:
once it goes GA you can't change the name though, but yeah it is not a big deal. good to leave as is